src/Files.App/Data/Commands/HotKey/HotKeyHelpers.cs

```diff
@@ -13,19 +13,21 @@ public static KeyModifiers GetCurrentKeyModifiers()
 		{
 			var modifiers = VirtualKeyModifiers.None;
 
-			if (IsPressed(VirtualKey.Menu))
+			if (IsPressed(VirtualKey.Menu) || IsPressed(VirtualKey.LeftMenu) || IsPressed(VirtualKey.RightMenu))
 				modifiers |= VirtualKeyModifiers.Menu;
-			if (IsPressed(VirtualKey.Control))
+			if (IsPressed(VirtualKey.Control) || IsPressed(VirtualKey.LeftControl) || IsPressed(VirtualKey.RightControl))
 				modifiers |= VirtualKeyModifiers.Control;
-			if (IsPressed(VirtualKey.Shift))
+			if (IsPressed(VirtualKey.Shift) || IsPressed(VirtualKey.LeftShift) || IsPressed(VirtualKey.RightShift))
```

Comment:
Is this modification necessary? I think `VirtualKey.Menu`, `VirtualKey.Control`, and `VirtualKey.Shift` are sufficient.

Reply:
That's how it reads in this document.
https://learn.microsoft.com/en-us/uwp/api/windows.system.virtualkey?view=winrt-22621

Reply:
Either is fine for me.
I'd like to add comments if we remove.

Reply:
I'm also fine with either. The only basis is this document, so if you want to modify the code just in case, I think it is fine.

Reply:
I'll keep as was then.